setup.py

```diff
@@ -9,6 +9,6 @@
 setup(
     name="Jinja2",
     version=version,
-    install_requires=["MarkupSafe>=1.1"],
+    install_requires=["MarkupSafe>=1.1", "TatSu>=4.4"],
```

Comment:
This can't be a required dependency, it must only be in `requirements/tests.in`.